merlin/main.py

```diff
@@ -929,6 +932,99 @@ def generate_diagnostic_parsers(subparsers: ArgumentParser) -> None:
         "Example: '--vars LEARN=path/to/new_learn.py EPOCHS=3'",
     )
 
+    # merlin detailed-status
+    detailed_status: ArgumentParser = subparsers.add_parser(
+        "detailed-status",
+        help="Display a task-by-task status of a study.",
+    )
+    detailed_status.set_defaults(func=query_status, detailed=True)
+    detailed_status.add_argument(
+        "spec_or_workspace", type=str, help="Path to a Merlin YAML spec file or a launched Merlin study"
+    )
+    detailed_status.add_argument(
+        "--dump", type=str, help="Dump the status to a file. Provide the filename (must be .csv or .json).", default=None
```

Comment:
Could default here be the same as the study name/timestamped variant, with some suffix maybe for the time said status gets dumped in case this gets called repeatedly?

Reply:
I'm not sure if I would have access to the name of the study at this point in the code since it's at the very top level of our codebase. In other words, I'm not sure if I'd be able to access the yaml file/output directory that the user passes in since at this point the arguments will not have been parsed yet. 

My thinking was that this would just default to `None` to represent that no dumping will take place unless the user specifically decides to dump the status.